webpack/JobInvocationDetail/index.js

```diff
@@ -65,42 +92,68 @@ const JobInvocationDetailPage = ({
   };
 
   return (
-    <PageLayout
-      header={description}
-      breadcrumbOptions={breadcrumbOptions}
-      searchable={false}
-    >
```

Comment:
Why removing PageLayout? can the code use `customBreadcrumbs` instead? just to make everything unified

Reply:
I removed it because I wanted to use the grid with different span for the title and buttons, but I agree that everything should be unified so I changed it to PageLayout, but there are some differences like different truncating of the title:

before:
![image](https://github.com/theforeman/foreman_remote_execution/assets/78563507/961ea487-8ea6-4ff1-8499-b2f7f3ad47c3)
after:
![image](https://github.com/theforeman/foreman_remote_execution/assets/78563507/e2aaaefd-8de6-409b-924a-6c40cd57f7e3)